doc/sphinx/language/gallina-extensions.rst

```diff
@@ -824,7 +819,7 @@ Sections create local contexts which can be shared across multiple definitions.
    Most commands, like :cmd:`Hint`, :cmd:`Notation`, option management, … which
    appear inside a section are canceled when the section is closed.
 
-.. cmd:: Let @ident := @term
+.. cmd:: Let @ident @def_body
 
    This command binds the value :token:`term` to the name :token:`ident` in the
    environment of the current section. The name :token:`ident` is accessible
```

Comment:
This description is not completely exhaustive / accurate. It talks about `term` but `term` (the body) may not be provided, in which case the proof mode is opened. I suggest writing instead "The command works like `Definition` except that the declared constant is local to the section. When the section is closed..."